Source/WebCore/platform/graphics/cocoa/GraphicsContextCocoa.mm

```diff
@@ -60,14 +60,20 @@
 // NSColor, NSBezierPath, and NSGraphicsContext calls do not raise exceptions
 // so we don't block exceptions.
 
-#if USE(APPLE_INTERNAL_SDK)
-#include <WebKitAdditions/GraphicsContextCocoa.mm>
-#else
 static RetainPtr<CGColorRef> grammarColor(bool useDarkMode)
 {
+#if ENABLE(MAC_CATALYST_GRAMMAR_CHECKING)
```

Comment:
Given there's no new API here, you can just get rid of this flag entirely.

Reply:
Wenson points out that the flag itself is used elsewhere. It should get renamed, and you *could* remove this particular use of it, but I don't feel strongly about that.

Reply:
Yeah, either way sounds good to me! Though, it might be cleaner to simply omit this flag here in particular (as Tim noted).